packages/backend/src/tokens/authenticateContext.ts

```diff
@@ -228,8 +227,11 @@ class AuthenticateContext {
 
 export type { AuthenticateContext };
 
-export const createAuthenticateContext = (
-  ...args: ConstructorParameters<typeof AuthenticateContext>
-): AuthenticateContext => {
-  return new AuthenticateContext(...args);
+export const createAuthenticateContext = async (
+  clerkRequest: ClerkRequest,
+  options: AuthenticateRequestOptions,
+): Promise<AuthenticateContext> => {
+  const cookieSuffix = options.publishableKey ? await getCookieSuffix(options.publishableKey) : '';
+  const ctx = new AuthenticateContext(cookieSuffix, clerkRequest, options);
```

Comment:
Same comment about moving cookieSuffix inside AuthenticateContext, but I don't feel strongly about this.

Reply:
closing as it's the same response to the above. Let's discuss if we want to use another approach and i will apply i to all your related comments.